print-battleground-state-changes

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python3
 
-import os
+import csv
 import collections
 import datetime
 import git
 import itertools
+import os
```

Comment:
nit: Unused import

Reply:
This was added for the caching, which uses `os.path`, and then I guess I sorted it into the right place wrt the other imports when I revisited the import list for the CSV generation.

Reply:
Oh, I'm blind. Sorry X.X

Reply:
No worries -- the unrelated hunk in this PR was definitely confusing.